### ISSUE TITLE
Page Regions

### ISSUE THREAD

**Issue by allanj-uaag** [w3c/low-vision-a11y-tf#64]
from https://lists.w3.org/Archives/Public/public-low-vision-comments/2016AprJun/0002.html
Perhaps a user need? or covered globally under 3.5.1

Different Look of Regions
I work with magnification software (Zoomtext) and find it very fine, when regions look different while using the Zoom Mode.

When I visit a website, I first try to get a visual and semantic overview without zooming. What is where on the site and how does it appear? I can’t read the text, but I see for instance, that the main navigation is on the left side with links on grey background. Complementary infos are on yellow background and the Content Info on the bottom is on light green background.

When zooming, I then have a better orientation, where I am going around. When for instance on the bottom of my zoom screen a light green area appears, I know that I have reached the end of the main content.

Websites, where everything looks the same are harder to navigate for me. This is also one of many reasons, why I don’t advise to use the Windows Contrast Mode: Everything looks the same and possibly helpful use of color for elements is no longer available.

**Comment by shawna-slh** [comment]
From [28 May 2016 telecon](http://www.w3.org/2016/05/26-lvtf-minutes.html):
- Also, some people have issues with stark changes.
- Also, being able to differentiate adjacent colors.

**Comment by allanj-uaag** [comment]
https://www.w3.org/2016/05/26-lvtf-minutes.html#item03

**Comment by shawna-slh** [comment]
I totally agree that this increases usability for all, and even more so for screen magnification users. I'm struggling with this being a low vision user need/requirement rather than a best practice.

I propose that it doesn't quite meet the threshold for including in this LV user needs doc _at this time_, and we mark it as a Best Practice. (Hopefully before too long we'll get to working more on Best Practice doc!) We can add it to the LV user needs doc later if warranted.

**Comment by lauracarlson** [comment]
+1 to marking as a best practice.
